crates/solvers-dto/src/auction.rs

```diff
@@ -56,6 +56,10 @@ pub struct Order {
     pub app_data: AppDataHash,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub flashloan_hint: Option<FlashloanHint>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub wrapper: Option<H160>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub wrapper_data: Option<Vec<u8>>,
```

Comment:
Maybe we can define a wrapper object (similar to FlashloanHint) with sub-properites target and data to keep the top level schema clean.

Also, is the plan that wrapper will eventually replace flashloans in app data?

Reply:
Not only does that keep the schema nicer to look at it also helps you to prevent inconsistent state.
Currently you could specify a `wrapper_data` without specifying a `wrapper` which does not make sense. If you have a separate struct you could do this:
```rust
wrapper: Option<Wrapper>
```
With
```rust
struct Wrapper {
    address: H160,
    additional_data: Option<Vec<u8>>,
}
```
That way you can only specify additional data if you also provide the associated wrapper contract address.